packages/core/src/MosaicClient.js

```diff
@@ -78,6 +78,13 @@ export class MosaicClient {
     return this;
   }
 
+  /**
+   * Prepare the client before start running queries. You may query additional
```

Comment:
Typo. Could change comment to state "Prepare the client before the query() method is called."

Reply:
Updated